guides/common/modules/proc_enabling-pull-based-transport-on-a-host.adoc

```diff
@@ -0,0 +1,37 @@
+[id="enabling-pull-based-transport-on-a-host_{context}"]
+= Enabling Pull-Based Transport on a Host
+
+Use the following procedure to migrate a host to pull-based transport.
+
+.Prerequisite
+* The host is registered to Katello.
+* Ensure you have enabled the {Team} {project-client-name} repository.
+
+.Procedure
+. Install the conversion tool:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+yum install katello-pull-transport-migrate
+----
+This installs `foreman_ygg_worker` and `yggdrasil` as a dependency and executes the migration.
+. To verify the installation was successful, run `yggdrasil status` and execute a remote execution job against the host using the script provider job.
+. To register your host in the pull mode, navigate to the advanced fields of the *Global Registration Form* and set `REX pull mode` to `Yes`.
```

Comment:
This is an odd step in this procedure. It's a prerequisite that the host is registered. This step is IMHO part of a different procedure: creating new hosts configured to use pull mode.